libraries/chain/asset_evaluator.cpp

```diff
@@ -107,6 +120,13 @@ void_result asset_create_evaluator::do_evaluate( const asset_create_operation& o
       FC_ASSERT( op.precision == op.bitasset_opts->short_backing_asset(d).precision );
    }
 
+   // Taker fees should be zero until activation of BSIP81
+   if(now <= HARDFORK_BSIP_81_TIME) {
+      FC_ASSERT(!op.common_options.extensions.value.taker_fee_percent.valid()
+                || *op.common_options.extensions.value.taker_fee_percent == 0,
+                "Simple maker-taker fees are not yet activated");
+   }
```

Comment:
I think we can call `check_asset_options_hf_bsip81()` function directly to avoid duplicate code.